editor/src/clj/internal/util.clj

```diff
@@ -147,9 +147,9 @@
   [pred m]
   (into {} (filter pred m)))
 
-(defn key-set
-  [m]
-  (set (keys m)))
+(defmacro key-set [m]
+  ;; TODO: Replace all calls so we can get rid of this macro.
```

Comment:
I wanted to move the `key-set` function to the `coll` module, because the `util` module depends on a lot of other modules. However, `util/key-set` is used in a bunch of places and I didn't want to complicate this PR further. I'll do a separate PR and change all calls to `util/key-set` to use `coll/key-set` instead later.